src/transformStateWithClones.js

```diff
@@ -8,6 +8,31 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const result = [];
+  const tempState = { ...state };
+
+  for (const action of actions) {
```

Comment:
You can destruct data from action
const { type, extraData, ... } = action

Reply:
I change this